nodebuilder/share/config.go

```diff
@@ -1,42 +1,67 @@
 package share
 
 import (
-	disc "github.com/celestiaorg/celestia-node/share/availability/discovery"
+	"errors"
+	"fmt"
+
+	"github.com/celestiaorg/celestia-node/share/availability/discovery"
+	"github.com/celestiaorg/celestia-node/share/availability/light"
 	"github.com/celestiaorg/celestia-node/share/p2p/peers"
 	"github.com/celestiaorg/celestia-node/share/p2p/shrexeds"
 	"github.com/celestiaorg/celestia-node/share/p2p/shrexnd"
 )
 
+var (
+	ErrNegativeInterval = errors.New("interval must be positive")
+)
+
 type Config struct {
-	// UseShareExchange is a flag toggling the usage of shrex protocols for blocksync.
+	Light     light.Parameters
```

Comment:
Let's call this Availability, as otherwise, the Light does not mean anything.

We should also make it LN only

cc @distractedm1nd, as I remember you found a hacky way to do this